scripts/snapshot.mjs

```diff
@@ -15,7 +15,8 @@ const featureFlags = [
   'vitest',
   'cypress',
   'playwright',
-  'nightwatch'
+  'nightwatch',
+  'devtools'
```

Comment:
This is going to make a _lot_ of new snapshots (it's a combination).
I think this should be removed, or maybe just added as single snapshot with the default config to check if it at least build without issues.

Reply:
Moved devtools as a separate flag. It will create a single new instance with base template & Vue DevTools
<img width="313" alt="Screenshot 2024-03-07 at 18 30 42" src="https://github.com/vuejs/create-vue/assets/69005114/3503105f-4a5d-477f-95aa-56241b18ad9e">